demo/public/monocle-client-angular.js

```diff
@@ -1,758 +0,0 @@
-/******/ (function(modules) { // webpackBootstrap
```

Comment:
This entire file is not git-ignored, and it's recreated when you run `npm run demo`